modules/yaml/yaml.go

```diff
@@ -35,13 +35,22 @@ func renderHorizontalHtmlTable(m yaml.MapSlice) string {
 		key := mi.Key
 		value := mi.Value
 
-		if key != nil && reflect.TypeOf(key).String() == "yaml.MapSlice" {
+		switch reflect.TypeOf(key).String() {
```

Comment:
I was originally using .String() as well. For this update I just added another if. Didn't know that then warrants a whole new rewrite. Will see if I can do this without the type being written as a string...new to this.